instalint-http-daemon/src/main/webapp/main.js

```diff
@@ -206,3 +196,61 @@ function analyze(store) {
         document.getElementById("result").style.opacity = 0.7;
     }
 }
+
+function lineOffsets(code) {
```

Comment:
Until we find a way to include the unit tests, let's include a link to the jsfiddle in a comment, so we can go back to it when we need to change something

Reply:
Done.